source/manual/government-changes.html.md

```diff
@@ -109,3 +109,48 @@ we can run a rake task:
 
 This will end all ministerial roles except the Prime Minister. You might have to
 do that one manually.
+
+## Machinery of Government Changes
+
+Following machinery of government changes, developers can help content
+designers make bulk changes to documents and users.
+
+### Bulk changing users
+
+There is a Rake task in Signon to change the organisation that a user belongs
+to in bulk:
+[`data_hygiene:bulk_update_organisation[csv_file]`][signon-bulk-update-organisation].
+
+It accepts a CSV file should have at least the following columns (in any
+order):
+
+- `Old email` - The old email address of the user.
+- `New email` - The new email address of the user.
+- `New organisation` - The slug of the new organisations for the user.
+
+If you want to bulk move some users across to a different organisation, follow
+the CSV file guidance and then Platform Health can do this for you.
+
+**Note:** It may be necessary to leave the old users around so they can still
+edit content for the old organisation. In that case you can bulk invite new
+users [using the interface in Signon][signon-batch-invitations].
+
+[signon-bulk-update-organisation]: https://github.com/alphagov/signon/blob/256ede9caa6061fd68b1daffd7123fff49df679f/lib/tasks/data_hygiene.rake#L3
+[signon-batch-invitations]: https://signon.publishing.service.gov.uk/batch_invitations/new
+
+### Bulk retagging documents
+
+There is a Rake task in Whitehall to change the lead and supporting
+organisations that a document is tagged to in bulk:
+[`data_hygiene:bulk_update_organisation[csv_file]`][whitehall-bulk-update-organisation].
+
+It accepts a CSV file should have at least the following columns (in any
+order):
+
+- `Slug` - The slug of the document.
```

Comment:
Can we make this a table? Or have the description delimited by colons? Same for the one above.

```suggestion
- `Slug`: the slug of the document.
```

Reply:
Yep, I've reformatted it as a table.